Tests/Form/Widget/FilterChoiceWidgetTest.php

```diff
@@ -90,7 +87,10 @@ protected function getChoiceClass()
      */
     protected function getDefaultOption()
     {
-        if (version_compare(Kernel::VERSION, '2.6.0', '>=')) {
+        if (method_exists(
+            'Symfony\Component\Form\Tests\AbstractLayoutTest',
+            'testSingleChoiceNonRequiredWithPlaceholder'
```

Comment:
Using a method on a test is a bad idea.

There are a lot of way it could :
 - If the autoloader is optimized (with `--optimize --classmap-authoritative`) because Test folders are excluded from classmap.
 - If someone remove tests folders for deployments
 - If the test change (they aren't covered by the BC promise)
 - If symfony start exluding tests from archives (https://github.com/symfony/symfony/pull/17995)